Report conversion fails when reading a method without a line number

Found in version 2.0.6.

When JacoboTask is reading a Jacoco report and encounters a method with no associated line number, it fails with this stacktrace:
```
Caused by: java.lang.NumberFormatException: For input string: ""
        at com.kageiit.jacobo.JacoboTask$_method_lines_closure3.doCall(JacoboTask.groovy:130)
        at com.kageiit.jacobo.JacoboTask.method_lines(JacoboTask.groovy:130)
...
```

Here's an excerpt from a Jacoco report file that illustrates the problem:
```xml
<method name="builder" desc="()Lcom/uber/mira/database/Region$RegionBuilder;" line="43">
<counter type="INSTRUCTION" missed="0" covered="4"/>
<counter type="LINE" missed="0" covered="1"/>
<counter type="COMPLEXITY" missed="0" covered="1"/>
<counter type="METHOD" missed="0" covered="1"/>
</method>
<method name="toBuilder" desc="()Lcom/uber/mira/database/Region$RegionBuilder;">
<counter type="INSTRUCTION" missed="0" covered="40"/>
<counter type="COMPLEXITY" missed="0" covered="1"/>
<counter type="METHOD" missed="0" covered="1"/>
</method>
```

Note that the "builder" method is tagged with "line=43", but the "toBuilder" method has no line number.  The latter will cause JacoboTask to fail.

It's worth noting that these methods were generated by Lombok's `@Builder` annotation (with `toBuilder = true`).  It's possible that the lack of line number is a bug in either Lombok or Jacoco, but in any case Jacobo should be able to handle it.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: groovy
 jdk:
-  - oraclejdk7
+  -  oraclejdk8
 env:
   - TERM=dumb
 script:

--- a/jacobo-plugin/src/main/groovy/com/kageiit/jacobo/JacoboTask.groovy
+++ b/jacobo-plugin/src/main/groovy/com/kageiit/jacobo/JacoboTask.groovy
@@ -126,6 +126,10 @@ class JacoboTask extends DefaultTask {
     }
 
     def static method_lines(method, methods, lines) {
+        if (method.@line.isEmpty()) {
+            return []
+        }
+
         def start_line = method.@line.toInteger()
         def larger = methods.findAll { it.@line.toInteger() > start_line }.collect {
             it.@line.toInteger()

